cloudcopy/server/api/v0/endpoints.py

```diff
@@ -1,25 +1,62 @@
-from adbc.store import Database as DatabaseDriver
+from fastapi import Depends
+from adbc.store import Database
 from cloudcopy.server.api import api
-from cloudcopy.server.config import settings
-from cloudcopy.server.models import Database
+from cloudcopy.server.api.utils import (
+    get_records,
+    get_record,
+    add_record,
+    edit_record,
+    delete_record,
+)
 from cloudcopy.server.storage import get_internal_database
-from .schemas import DatabaseSchema, AddDatabaseSchema
-from fastapi import Depends
+from .schemas import (
+    GetDatabaseOut,
+    GetDatabasesOut,
+    AddDatabaseOut,
+    AddDatabaseIn,
+    EditDatabaseIn,
+    EditDatabaseOut,
+    SetDatabaseIn,
+    SetDatabaseOut,
+    DeleteDatabaseOut
+)
+
+version = 'v0'
+model = 'database'
+endpoint = 'databases'
+
+
+@api.get(f"/{version}/{endpoint}/", response_model=GetDatabasesOut)
```

Comment:
Endpoint boilerplate: FastAPI decorator with `response_model` + function with a `Depends` on call to get internal DB (singleton)